src/runtime_src/driver/xclng/tools/xbutil/xbutil.h

```diff
@@ -600,7 +600,6 @@ class device {
         lines.push_back(ss.str());
     }
 
-
     int readSensors( void ) const
     {
         sensor_tree::put( "version",                 "1.1.0" ); // json schema version
```

Comment:
Remove this line, since you're inserting schema version in baseInit().

Reply:
Done